js/levelsSuperhero.js

```diff
@@ -0,0 +1,25 @@
++// defined in /js/tiles.js
+ +/* global NullTile */
+ +/* global door */
+ +/* global PlayerStartsAt */
+ +
+ +// defined in /js/level.js
+ +/* global Level */
+ +
+ +// createTestLevel is used in /js/gui.js ignoring the error
+ +// ignore JSHintBear
+ +function createTestLevel()
```

Comment:
Please add this code to to levels.js and rename the function to createSuperheroLevel()